src/lib/cleaver/SizingFieldCreator.cpp

```diff
@@ -489,23 +462,16 @@ namespace cleaver
     double min, temp;
     if (oct->a.index[0] > oct->b.index[0] || oct->a.index[1] > oct->b.index[1] || oct->a.index[2] > oct->b.index[2])
     {
-      //printf("entering %d %d %d %d %d %d %d\n", stack, oct->a.index[0], oct->a.index[1], oct->a.index[2], oct->b.index[0], oct->b.index[1], oct->b.index[2]);
```

Comment:
this print statements are usually best to convert to log calls, if cleaver has something it can use